src/makeRobot.js

```diff
@@ -38,7 +38,87 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    version: `chip version: ${version}`,
```

Comment:
Why did you write this way? Better leave here simply 
```suggestion
    version,
```